prev_processed_file.py

```diff
@@ -16,9 +16,9 @@ def __init__(self, file_path) -> None:
         self.file_path = file_path
         self.__processed_restaurants = self.__input_processed_companies()
 
-    def get_by_pnr(self, pnr: str) -> dict:
+    def get_control_date_by_pnr(self, pnr: str) -> dict:
```

Comment:
Change return type hint to str. I made the change to index on `prev_processed_restaurant` in #13 because I expected a dict from here.